layouts/partials/task/card-summary.html

```diff
@@ -1,21 +1,71 @@
-{{ with .Site.Params.taskData}}
-    {{$url := .}}
-    {{$data := getJSON $url}}
-    {{range $data}}
-    <div class="card">
-       <img src="" class="card-img-top">   
+<div id="noselection">
+    <img src="" alt="Select a task to begin">
+</div>
+{{$URLpathtask := path.Join .Site.Params.api.baseid .Params.table .Params.view}}
+{{ $airtable := getJSON .Site.Params.api.name $URLpathtask (dict "Authorization" .Site.Params.api.key)  }}
+{{$URLpathuser := path.Join .Site.Params.api.baseid "Users" "?view=Master%20View"}}
+{{ $airtableuser := getJSON .Site.Params.api.name $URLpathuser (dict "Authorization" .Site.Params.api.key)  }}
 
-    <div class="card-body">
-        <h2>{{.TaskName}}</h2>
-        <p class="card-text">
-            <ul>
-                <li>{{.Location}}</li>
-                <li>{{.DateForCompletion}}</li>
+
+
+<div class="text-center">{{ .Content }}</div>
+{{range $airtable.records }}
+<div id="modal-container-{{.id}}" class="modal-container">
+  <div class="modal-back">
+  {{with .fields}}
+    <div class="modal">
+        <div class="row">
+            <div class="col-md-10">
+            <h2>{{.TaskName}}</h2>
+            <ul class="task-specifics">
+                {{$user := $airtableuser.records}}
+                {{with .Creator}}
+                {{$creator:=index . 0}}
+                {{range where $user "id" $creator}}
+                <li>Posted by {{.fields.Name}} </li>
+                {{end}}
+                {{end}}
+                <li>📍 {{.Location }}</li>
+                <li>🗓 {{time.Format "Monday, Jan 2, 2006" .DateForCompletion}}</li>
+                {{with .Attachments}}
+                <img src="{{index . 0 "url"}}" width="100%">
+                {{end}}
             </ul>
-        </p>
-        <span class="badge badge-primary">{{.Status}}</span>
-    </div>
-    <h3>{{.Budget}}</h3>
+            </div>
+            <div class="col-md-4">
+                <h5>Task Budget</h5>
+                <h3>${{.Budget}}</h3>
+            </div>
+        </div>
+        <h3>Description of task:</h3>
+        <p>{{.DescriptionOfTask}}</p>
+        <h2>Offers</h2>
+        {{/*  {{ partial "task/offers.html" .}}  */}}
+
     </div>
-    {{end}}
-{{end}}
\ No newline at end of file
+  {{end}}
+  </div>
+</div>
+{{end}}
+<script>
```

Comment:
Am proud of this script, and will leave it here for now as am using Hugo to inject data into an array